0-setup-libraries.ipynb

```diff
@@ -30,46 +29,2868 @@
+      "Extracting : certifi-2020.4.5.1-py37_0.conda:  46%|████▌     | 16/35 [00:13<02:41,  8.53s/it]\r",
+      "Extracting : chardet-3.0.4-py37_1003.conda:  49%|████▊     | 17/35 [00:13<02:33,  8.53s/it]  \r",
+      "Extracting : idna-2.9-py_1.conda:  51%|█████▏    | 18/35 [00:13<02:24,  8.53s/it]          \r",
+      "Extracting : pycosat-0.6.3-py37h7b6447c_0.conda:  54%|█████▍    | 19/35 [00:13<02:16,  8.53s/it]\r",
+      "Extracting : pycparser-2.20-py_0.conda:  57%|█████▋    | 20/35 [00:13<02:07,  8.53s/it]         \r",
+      "Extracting : pysocks-1.7.1-py37_0.conda:  60%|██████    | 21/35 [00:13<01:59,  8.53s/it]\r",
+      "Extracting : ruamel_yaml-0.15.87-py37h7b6447c_0.conda:  63%|██████▎   | 22/35 [00:13<01:50,  8.53s/it]\r",
+      "Extracting : six-1.14.0-py37_0.conda:  66%|██████▌   | 23/35 [00:13<01:42,  8.53s/it]                 \r",
+      "Extracting : tqdm-4.46.0-py_0.conda:  69%|██████▊   | 24/35 [00:13<01:33,  8.53s/it] \r",
+      "Extracting : cffi-1.14.0-py37he30daa8_1.conda:  71%|███████▏  | 25/35 [00:13<01:25,  8.53s/it]\r",
+      "Extracting : conda-package-handling-1.6.1-py37h7b6447c_0.conda:  74%|███████▍  | 26/35 [00:13<01:16,  8.53s/it]\r",
+      "Extracting : setuptools-46.4.0-py37_0.conda:  77%|███████▋  | 27/35 [00:13<01:08,  8.53s/it]                   \r",
+      "Extracting : cryptography-2.9.2-py37h1ba5d50_0.conda:  80%|████████  | 28/35 [00:13<00:59,  8.53s/it]\r",
+      "Extracting : wheel-0.34.2-py37_0.conda:  83%|████████▎ | 29/35 [00:13<00:51,  8.53s/it]              \r",
+      "Extracting : pip-20.0.2-py37_3.conda:  86%|████████▌ | 30/35 [00:13<00:42,  8.53s/it]  \r",
+      "Extracting : pyopenssl-19.1.0-py37_0.conda:  89%|████████▊ | 31/35 [00:13<00:34,  8.53s/it]\r",
+      "Extracting : urllib3-1.25.8-py37_0.conda:  91%|█████████▏| 32/35 [00:13<00:25,  8.53s/it]  \r",
+      "Extracting : requests-2.23.0-py37_0.conda:  94%|█████████▍| 33/35 [00:13<00:17,  8.53s/it]\r",
+      "Extracting : conda-4.8.3-py37_0.tar.bz2:  97%|█████████▋| 34/35 [00:13<00:08,  8.53s/it]  \r",
+      "                                                                                        \r"
      ]
     }
    ],
    "source": [
     "%%bash\n",
     "#!/bin/bash\n",
     "\n",
-    "CMSSW_VER=\"CMSSW_10_6_13\"\n",
-    "KERNEL_NAME=\"machine-learning-hats\"\n",
+    "KERNEL_NAME=\"machine-learning-hats-2020\"\n",
     "\n",
     "set -e\n",
-    "# Get the CMSSW libraries (specifically ROOT)\n",
-    "SCRAM_ARCH=$(ls -d /cvmfs/cms.cern.ch/$(/cvmfs/cms.cern.ch/common/cmsos)*/cms/cmssw/${CMSSW_VER} | tail -n 1 | awk -F / '{ print $4 }')\n",
-    "export SCRAM_ARCH\n",
-    "source /cvmfs/cms.cern.ch/cmsset_default.sh\n",
-    "if [ ! -d cmssw-env ]; then\n",
-    "    scramv1 project --name cmssw-env CMSSW $CMSSW_VER \n",
-    "fi\n",
-    "cd cmssw-env\n",
-    "eval `scramv1 runtime -sh`\n",
+    "wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O $HOME/miniconda3.sh\n",
```

Comment:
Add `--progress=dot:giga` to the wget command so that it doesn't print so many lines to the progress bar.

Reply:
Fixed.